Source/MultipartFormData.swift

```diff
@@ -55,7 +55,10 @@ open class MultipartFormData {
         }
 
         static func randomBoundary() -> String {
-            String(format: "alamofire.boundary.%08x%08x", arc4random(), arc4random())
+            let first = UInt32.random(in: UInt32.min...UInt32.max)
+            let second = UInt32.random(in: UInt32.min...UInt32.max)
+
+            return String(format: "alamofire.boundary.%08x%08x", first, second)
```

Comment:
Random thought: when I reimplemented this using `UInt32.random`, I kept the original logic of two different integers getting formatted, but it sure seems like this could be _one_ integer that gets formatted twice as wide.

Reply:
Hmm, never mind. There’s not a built-in format specifier for a 64-bit integer in hex format.